ci/build-client.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
should we make `build-client` and `build-gamelogic` parameterizable so that users could build the workers for whichever `buildTarget` they want

Reply:
Done, they now accept a parameter

Reply:
would it make sense to have another parameter to be "UnityClient" or "UnityGameLogic" to prevent code duplication?

Reply:
get `ci/build.sh` back and remove `ci/build-client.sh` and `ci/build-gamelogic.sh` xD

would prefer that actually as it also makes it easier to add more worker types later

Reply:
I think we should do this work once we have more than 2 workers - we may have a better design once we have a bigger set of example use cases.

Reply:
I would really prefer to do this now as this seems far more error-prone. e.g. just accidentally changing one script and not the other